README.md

```diff
@@ -713,46 +713,47 @@ classDiagram
     class Comment {
     }
 
-    %% end of the generated part
+    RuleSet <|-- DeclarationBlock: inheritance
+    Renderable <|.. RuleSet: realization
+    Commentable <|.. RuleSet: realization
+    RuleSet <|-- AtRuleSet: inheritance
+    AtRule <|.. AtRuleSet: realization
+    Selector <|-- KeyframeSelector: inheritance
+    Renderable <|-- AtRule: inheritance
+    Commentable <|-- AtRule: inheritance
+    AtRule <|.. Charset: realization
+    AtRule <|.. Import: realization
+    AtRule <|.. CSSNamespace: realization
+    Renderable <|.. Rule: realization
+    Commentable <|.. Rule: realization
+    SourceException <|-- OutputException: inheritance
+    UnexpectedTokenException <|-- UnexpectedEOFException: inheritance
+    Exception <|-- SourceException: inheritance
```

Comment:
Should inheritance from in-built classes be included?

Reply:
As far as I know, there is no option to configure this behavior. So if we want to automate this, I propose we stick to what the tool generates.